S4TF_Tutorial_3.ipynb

```diff
@@ -0,0 +1,1772 @@
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 47040016 (45M) [application/octet-stream]\n",
+            "Saving to: ‘train-images-idx3-ubyte?raw=true’\n",
+            "\n",
+            "train-images-idx3-u 100%[===================>]  44.86M   289MB/s    in 0.2s    \n",
+            "\n",
+            "2019-06-11 02:45:59 (289 MB/s) - ‘train-images-idx3-ubyte?raw=true’ saved [47040016/47040016]\n",
+            "\n",
+            "downloading ...  train-labels-idx1-ubyte\n",
+            "--2019-06-11 02:45:59--  https://github.com/Ayush517/S4TF-Tutorials/blob/master/fashion-MNIST/train-labels-idx1-ubyte?raw=true\n",
+            "Resolving github.com (github.com)... 140.82.118.3\n",
+            "Connecting to github.com (github.com)|140.82.118.3|:443... connected.\n",
+            "HTTP request sent, awaiting response... 302 Found\n",
+            "Location: https://github.com/Ayush517/S4TF-Tutorials/raw/master/fashion-MNIST/train-labels-idx1-ubyte [following]\n",
+            "--2019-06-11 02:45:59--  https://github.com/Ayush517/S4TF-Tutorials/raw/master/fashion-MNIST/train-labels-idx1-ubyte\n",
+            "Reusing existing connection to github.com:443.\n",
+            "HTTP request sent, awaiting response... 302 Found\n",
+            "Location: https://raw.githubusercontent.com/Ayush517/S4TF-Tutorials/master/fashion-MNIST/train-labels-idx1-ubyte [following]\n",
+            "--2019-06-11 02:45:59--  https://raw.githubusercontent.com/Ayush517/S4TF-Tutorials/master/fashion-MNIST/train-labels-idx1-ubyte\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 60008 (59K) [application/octet-stream]\n",
+            "Saving to: ‘train-labels-idx1-ubyte?raw=true’\n",
+            "\n",
+            "train-labels-idx1-u 100%[===================>]  58.60K  --.-KB/s    in 0.007s  \n",
+            "\n",
+            "2019-06-11 02:46:00 (8.53 MB/s) - ‘train-labels-idx1-ubyte?raw=true’ saved [60008/60008]\n",
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "LbPfmnDJ_o0e",
+        "colab_type": "code",
+        "outputId": "aa22254a-284a-4ebf-b598-403544fea4e3",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 187
+        }
+      },
+      "source": [
+        "let filemgr = FileManager.default\n",
+        "let filelist = try filemgr.contentsOfDirectory(atPath: \".\")\n",
```

Comment:
I'd probably simplify this by removing the `let filemgr` line and just doing `FileManager.default.contentsOfDirectory...` here. Also the indentation of the trailing curly bracket just below here should probably be moved to the left to be aligned properly.